pkg/sql/plan/function/builtins.go

```diff
@@ -174,6 +174,25 @@ var builtins = map[int]Functions{
 			},
 		},
 	},
+	UUID: {
+		Id: UUID,
+		TypeCheckFn: func(_ []Function, inputs []types.T) (overloadIndex int32, ts []types.T) {
+			if len(inputs) == 0 {
+				return int32(0), nil
+			}
+			return wrongFunctionParameters, nil
+		},
+		Overloads: []Function{
+			{
+				Index:     0,
+				Flag:      plan.Function_STRICT,
+				Layout:    STANDARD_FUNCTION,
+				Volatile:  true,
+				Args:      []types.T{types.T_int64},
```

Comment:
maybe use T_any will be better。because this parameter make any sense

Reply:
OK， I will try it.